usecases/How_to_Setup_a_Domain_Whitelabel.md

```diff
@@ -0,0 +1,5 @@
+# How to Setup a Domain Whitelabel
```

Comment:
nit: directory name should be `use_cases`

Reply:
Also it's no longer called domain whitelabeling: https://github.com/sendgrid/sendgrid-java/pull/544